ui/desktop/src/main.ts

```diff
@@ -697,9 +706,10 @@ app.whenReady().then(async () => {
   });
 });
 
-// Quit when all windows are closed, except on macOS.
+// Quit when all windows are closed, except on macOS or if we have a tray icon.
 app.on('window-all-closed', () => {
-  if (process.platform !== 'darwin') {
+  // Only quit if we're not on macOS or don't have a tray icon
+  if (process.platform !== 'darwin' || !tray) {
```

Comment:
When do we ever not create the tray icon?

Reply:
I wanted to play it safe